CppSamples/Maps/DisplayDeviceLocation/DisplayDeviceLocation.cpp

```diff
@@ -71,10 +73,22 @@ void DisplayDeviceLocation::componentComplete()
 
 void DisplayDeviceLocation::startLocationDisplay()
 {
-  //! [start location display api snippet]
-  // turn on the location display
-  m_mapView->locationDisplay()->start();
-  //! [start location display api snippet]
+  QLocationPermission locationPermission{};
```

Comment:
Per https://doc.qt.io/qt-6.5/qlocationpermission.html#details, the default QLocationPermission requests approximate location only. Does this not matter and you get all options presented anyway? I would be more explicit about the type of access requested if possible.

Reply:
Good point, thanks for pointing it out. I changed it to `Precise` accuracy and `WhenInUse` availability which I think makes the most sense for this app.

Reply:
It is strange though, it doesn't seem to matter what I put for `Accuracy` and `Availability`. The device will display all options relating to precision and availability regardless of what options I have set on the permissions object.